app/gui/src/dashboard/modals/AcceptInvitationModal.tsx

```diff
@@ -0,0 +1,61 @@
+/** @file Modal for handling user organization invitation. */
+
+import { Alert } from '#/components/Alert'
+import { AlertDialog } from '#/components/AlertDialog'
+import { Text } from '#/components/Text'
+import { backendMutationOptions } from '#/hooks/backendHooks'
+import type * as backend from '#/services/Backend'
+import { useBackends } from '$/providers/backends'
+import { useText } from '$/providers/react'
+import { useMutation } from '@tanstack/react-query'
+import { toast } from 'react-toastify'
+
+/** Props for a {@link AcceptInvitationModal}. */
+export interface AcceptInvitationModalProps {
+  readonly invitation: backend.Invitation
+}
+
+/** Modal for handling subscription after the trial period ended. */
+export function AcceptInvitationModal(props: AcceptInvitationModalProps) {
+  const { invitation } = props
+  const { organizationId, organizationName, userEmail } = invitation
+  const { getText } = useText()
+  const { remoteBackend } = useBackends()
+  const onConfirm = useMutation(
+    backendMutationOptions(remoteBackend, 'updateUser', {
+      meta: { invalidates: [['updateUser']], awaitInvalidates: true },
```

Comment:
these should go in `INVALIDATION_MAP` if they aren't already, instead of being explicitly specified every time they need to be used

Reply:
`updateUser: [INVALIDATE_ALL_QUERIES],` already have invalidate all so should be good